spec/requests/organisations_controller_spec.rb

```diff
@@ -279,6 +279,88 @@
             expect(page).to have_content("To report a merge or update your organisation details, ")
             expect(page).to have_link("contact the helpdesk", href: "https://dluhcdigital.atlassian.net/servicedesk/customer/portal/6/group/11")
           end
+
+          it "does not display merge history if there is none" do
+            expect(page).not_to have_content("View all organisations that were merged into #{organisation.name}")
+          end
+
+          context "when the organisation has absorbed other organisations" do
+            let!(:absorbed_organisation) { create(:organisation, name: "First Absorbed Organisation") }
+            let!(:other_absorbed_organisation) { create(:organisation, name: "Other Absorbed Organisation") }
+            let!(:old_absorbed_organisation) { create(:organisation, name: "Old Absorbed Organisation") }
+
+            before do
+              absorbed_organisation.update!(merge_date: Time.zone.local(2023, 4, 3), absorbing_organisation: organisation)
+              other_absorbed_organisation.update!(merge_date: Time.zone.local(2023, 4, 3), absorbing_organisation: organisation)
+              old_absorbed_organisation.update!(merge_date: Time.zone.local(2023, 4, 2), absorbing_organisation: organisation)
+              get "/organisations/#{organisation.id}/details", headers:, params: {}
+            end
+
+            it "displays a list of absorbed organisations" do
+              expect(page).to have_content("View all organisations that were merged into #{organisation.name}")
+              expect(page).to have_content("Merge date: 3 April 2023")
+              expect(page).to have_content("First Absorbed Organisation")
+              expect(page).to have_content("Other Absorbed Organisation")
+              expect(page).not_to have_content("Old Absorbed Organisation")
```

Comment:
This seems to contradict the phrasing "View all organisations...", but I see it's in the designs - may be worth checking in with Olivia?

Reply:
Good point, will check!